src/locale/sv.js

```diff
@@ -0,0 +1,34 @@
+  accepted             : 'Fält :attribut måste accepteras.',
```

Comment:
You are missing the opening and closing lines of the object, check this file as an example: https://github.com/dockwa/simple-react-validator/blob/master/src/locale/el.js#L1-L2